src/renderers/dom/shared/dangerousStyleValue.js

```diff
@@ -48,7 +51,27 @@ function dangerousStyleValue(name, value) {
   }
 
   if (typeof value === 'string') {
-    value = value.trim();
+    if (__DEV__) {
+      if (component) {
+        var owner = component._currentElement._owner;
+        var componentName = owner ? owner.getName() : component._currentElement.type;
```

Comment:
Is this line correct?  It is conflating the owner name with the element type, which feels wrong to me.

I think we actually do want to print both pieces of information (the owner if it exists, and the current element's tag) in the error message.  Something like `a %s tag (owner: %s) was passed a numeric string value for CSS property `%s` (value: %s) which will be treated as a unitless number in a future version of React.`